setup_dev_env.sh

```diff
@@ -34,9 +34,9 @@ if [ ! -e $conda_bin ]; then
 fi
 
 # define a python version to initialise the conda environment.
-# by default we assume python 3.7.
-python_version=${PYTHON_VERSION:-"3.7"}
-pytorch_version=${PYTORCH_VERSION:-"1.10.2"}
+# by default we assume python 3.8
+python_version=${PYTHON_VERSION:-"3.8"}
+pytorch_version=${PYTORCH_VERSION:-"1.13.0"}
 pytorch_mode=${PYTORCH_MODE:-""}  # use `cpuonly` for CPU or leave it in blank for GPU
```

Comment:
i'm uncertain if is necessary, but now have the `pytorch-cuda`

Reply:
is that a project ?

Reply:
I really don't know, I just noticed that it is on the official recommendation page -- https://anaconda.org/pytorch/pytorch-cuda

![image](https://user-images.githubusercontent.com/20444345/207052535-2cac6b1b-71e2-474e-a1e0-af2dfdb8f316.png)